src/graphql/schemas/custom/commonInputs.graphql

```diff
@@ -0,0 +1,194 @@
+scalar xhtml
+scalar positiveInt
+"""
+Base StructureDefinition for CodeableConcept Type: A concept that may be defined
+by a formal reference to a terminology or ontology or may be provided by text.
+"""
+input CodeableConcept_Input {
```

Comment:
can we consistent on naming?  this one has upper case "I" in "i"

Reply:
These types were copied from assymetirk's graphql fhir server, they are autogenerated as far as i know in that app, so i was reluctant to make changes there.

Reply:
Found the one inconsistency and fixed it, it must be a flaw in autogeneration but I will keep an eye out for that going forward.